numba/tests/support.py

```diff
@@ -266,6 +266,16 @@ def assertRefCount(self, *objects):
                 self.fail("Refcount changed from %d to %d for object: %r"
                           % (old, new, obj))
 
+    def assertRefCountEqual(self, *objects):
+        gc.collect()
+        rc = [sys.getrefcount(x) for x in objects]
+        rc_0 = rc[0]
+        for i in range(len(objects))[1:]:
```

Comment:
```suggestion
        for i in range(1, len(objects)):
```
is it just this?

Reply:
yup. you can imagine `range()` returning a list. that's allowed use from py2 when range() does return a list.

Reply:
Can leave it, it's just a preference.